### PR TITLE
changing method of migration for UserRewardInfo

### PR DESCRIPTION
**Pull request by avkr003**

### CODE REVIEW

**Comment by avkr003** [comment]
This PR does not migrate UserRewardInfos to optimized keys due to software test upgrade failing

**Comment by amityadav0** [comment]
Because we have a lot of migration already in this upgrade, is that could be the reason ?

**Comment by avkr003** [comment]
Don't think so. We are not changing the data structure but just the keys. In my manual tests, it worked.

File: x/masterchef/keeper/user_reward_info.go [hunk 1]
```diff
@@ -47,23 +53,25 @@ func (k Keeper) GetAllUserRewardInfos(ctx sdk.Context) (list []types.UserRewardI
...
-
-	for ; iterator.Valid(); iterator.Next() {
-		var val types.UserRewardInfo
-		k.cdc.MustUnmarshal(iterator.Value(), &val)
-		list = append(list, val)
-	}
-
-	return
-}
+//func (k Keeper) MigrateFromV2UserRewardInfos(ctx sdk.Context) {
```
Comment (amityadav0) [inline x/masterchef/keeper/user_reward_info.go:62]:
```suggestion
// TODO: Optimise getuserrewardsinfo
//func (k Keeper) MigrateFromV2UserRewardInfos(ctx sdk.Context) {
```

**Review by amityadav0 — APPROVED** [review]

**Review by cosmic-vagabond — APPROVED** [review]

**Pull request merged**